### PR TITLE
[DCCsi] Maya, config, settings and bootstrapping update, to start Maya 

### PR DESCRIPTION
**Pull request by HogJonny-AMZN**
Signed-off-by: Jonny Galloway <gallowj@amazon.com>

## What does this PR do?
1. Some updates to the legacy .bat env
2. Moves some maya launch .bat files into < dccsi >\Tools\DCC\Maya\
3. Adds complete < dccsi >\Tools\DCC\Maya\constants.py (mirrors legacy .bat env)
4. Adds a working < dccsi >\Tools\DCC\Maya\config.py
5. Config can generate settings.local.json (for dev to configure/override settings)
6. Adds a working  < dccsi >\Tools\DCC\Maya\start.py
7. Adds maya to dccsi menu framework to start from O3dE Editor menu

(Future PR):
- add cli support to set maya version to start
-  set cli options to start mayapy.exe or mayabatch instead?

## How was this PR tested?
1. The start.py module can be run in IDE (wing) and verified to start Maya
2. The start.py module can be run from commandline and verified to start maya
3. User can start Maya from the O3DE Editor Menus

## Requirements
- Windows only (for now, still early)
- Maya 2022, or 2023 w\ python 3 must be installed (we recommend the default install path)
- ^ the default is Maya 2023
- You must ensure the package dependencies are installed against mayapy runtime
1. open CMD
2. > cd C:\path\to\o3de\Gems\AtomLyIntegration\TechnicalArt\DccScriptingInterface\
3. > python foundation.py -py "C:\Program Files\Autodesk\Maya2023\bin\mayapy.exe"
-  If you need to alter the version, the MAYA_LOCATION (install path), or any other settings in the configuration, you will need to generate the settings.local.json, which can then be edited in a text editor
1. open CMD
2. > cd C:\path\to\o3de\Gems\AtomLyIntegration\TechnicalArt\DccScriptingInterface\
3. > python Tools\DCC\Maya\config.py

### CODE REVIEW

**Comment by HogJonny-AMZN** [comment]
Running the maya config.py module from the commandline
image
image

**Comment by HogJonny-AMZN** [comment]
Example of generated settings.local.json (dev and edit this, to persist local settings into the environment and configuration ... set/override behavior)
[settings.local.zip](https://github.com/o3de/o3de/files/9749865/settings.local.zip)

**Comment by HogJonny-AMZN** [comment]
Starting Maya from the cli
1. maya gui
2. maya console with validation tool bootstrapping occured

image

File: Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings [hunk 1]
```diff
@@ -173,17 +173,17 @@
...
             //Reference
             "_ref": [ "ReferenceImage" ],
             "_refhdr": [ "ReferenceImage_HDRLinear" ],
             "_refhdru": [ "ReferenceImage_HDRLinearUncompressed" ],
             "_reflinear": [ "ReferenceImage_Linear" ],
-            
+
             // Reflectance, linear single channel
-			"_cavity": [ "Reflectance" ],
+                "_cavity": [ "Reflectance" ],
```
Comment (cgalvan) [inline Gems/Atom/Asset/ImageProcessingAtom/Assets/Config/ImageBuilder.settings:186]: Not sure if it's just how it's being rendered in github diff but some of the spacing here looks off

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Registry/AssetProcessorGemConfig.setreg [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "O3DE": {
+        "Atom": {
+            "RPI": {
+                "MaterialTypeBuilder": {
+                    "CreateAllPropertiesMaterial": true
+				}
```
Comment (cgalvan) [inline Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Registry/AssetProcessorGemConfig.setreg:7]: Nit: 
```suggestion
		}
```

**Review by cgalvan — APPROVED** [review]

**Review by shawstar — APPROVED** [review]

**Review by gadams3 — APPROVED** [review]

**Pull request merged**